src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -704,12 +705,16 @@ void prepare_primitive_fusing::fuse_simple_primitives(program &p) {
             if (_lo.get_optimization_attributes().use_onednn_impls) {
                 if (input.is_type<reshape>() || input.is_type<concatenation>())
                     return;
-
+                if (activation_func == cldnn::activation_func::relu_negative_slope &&
+                    (input.is_type<fully_connected>() || input.is_type<gemm>())) {
+                    // prelu fusion is not implemented in oneDNN3.1 (CVS-108233)
+                    return;
+                }
                 // Activation should not be fused if oneDNN does NOT support it
                 if (_lo.is_primitive_implemented_for_onednn(input))  {
                     #ifdef ENABLE_ONEDNN_FOR_GPU
                     try {
-                        onednn::convert_activation_func(activation_node.get_primitive()->activation_function);
+                        onednn::convert_activation_func(activation_func);
```

Comment:
(random spot) could you add a unit test?

Reply:
Do you mean check the segfaul unit-test, correct?

Reply:
yes, segfault before this PR, OK after this PR.

Reply:
Applied